src/storage/blockchain/onchain-issuer-adapter/non-merklized/version/v0.0.1/types/NonMerklizedIssuerBase.ts

```diff
@@ -0,0 +1,617 @@
+/* Autogenerated file. Do not edit manually. */
```

Comment:
Do we really need this autogenerated file in sdk ?

Reply:
I suggest removing all autogenerated files and keeping only the .abi files with functions that are actively used in our services. This will simplify the PR, making it easier to review, and ensure alignment with our standard approach to interacting with Smart Contracts.